plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -773,8 +773,17 @@ public void createSchema(ConnectorSession session, String schemaName, Map<String
     }
 
     @Override
-    public void dropSchema(ConnectorSession session, String schemaName)
+    public void dropSchema(ConnectorSession session, String schemaName, boolean cascade)
     {
+        if (cascade) {
+            for (SchemaTableName viewName : listViews(session, Optional.of(schemaName))) {
+                dropView(session, viewName);
```

Comment:
Sorry not clear on the expected behavior, one qn: Should this catch the `ViewNotFoundException` or `TableNotFoundException`s and ignore it?